resources/migrations/001_update_migrations.yaml

```diff
@@ -5598,6 +5598,18 @@ databaseChangeLog:
             sql: >-
               DROP VIEW IF EXISTS v_content;
 
+  - changeSet:
+      id: v50.2024-03-02T17:41:00
+      author: qnkhuat
+      comment: Add metabase_table.properties
+      changes:
+        - addColumn:
+            tableName: metabase_table
+            columns:
+              - column:
+                  name: properties
```

Comment:
I'm curious how much thought has been given to the name "properties" - being so generic, are we intentionally leaving it open for ad hoc future use cases? I personally like to error towards narrow and specific, as one can always add further columns. What do you think about calling it something like `statistics`?

Reply:
I indeed wanted this to be generic, IMO optional fields like `show_in_getting_started` or `database_require_filter` could fall into this category. 

It actually started as Cam suggested me to put `database_require_filter` into a json column like this in the first place, and I think that's a good idea :)

Reply:
If we explicitly want this to be a wide assortment the name is fine I guess. I'm slightly allergic to generic names but don't have a counter suggestion.